File: nine.js
function nine(n){
    for (i = 0; i < 500; i++){
        for (j = 0; j < 500; j++){
            for (k = 0; k < 500; k++ ){
                if (i + j + k == n){
                    if (i**2 + j**2 == k**2){
                        return i*k*j
                    }
                }
            }
        }
    }
}

console.log(nine(1000))
//prints 3187500

File: nine.go
package main

import(
	"fmt"
	"math"
)

func main () {
	var counter int = 0
	var primer int = 1

	for counter < 10001 {
		primer++
		if isPrime(primer){
			counter++
		}
	}
	fmt.Println(primer)
}

func isPrime(natural int) bool {
	var limit int = int(math.Sqrt(float64(natural)))
	for i := 2; i < limit + 1; i++ {
		if natural % i == 0{
			return false
		}
		}
		return true
	}

//prints 104743